tools/project_manager/handle_bad_page.php

```diff
@@ -193,7 +193,7 @@ function show_resolution_form($projectid, $image, $state, $project_round, $is_a_
         }
 
         if (!empty($b_code)) {
-            echo "<b>" . _("Reason") . ":</b> {$PAGE_BADNESS_REASONS[$b_code]}</br>";
+            echo "<b>" . _("Reason") . ":</b> {$PAGE_BADNESS_REASONS[$b_code]["string"]}</br>";
```

Comment:
This is getting a little convoluted and I'm surprised PHP isn't complaining about using `"` inside a string delimited by `"`s. Perhaps bypass it altogether with:

```suggestion
            echo "<b>" . _("Reason") . ":</b> " . $PAGE_BADNESS_REASONS[$b_code]["string"] . "</br>";
```